src/commandHandlers/settingsCommandHandler.ts

```diff
@@ -29,4 +32,20 @@ export default class SettingsCommandHandler extends BaseCommandHandler {
     public async SwitchVstsAcc(): Promise<void> {
         await new Settings.SwitchVstsAccount(this.getCommandParams()).runNoClient();
     }
+
+    public async ShowStatusBar(): Promise<void> {
```

Comment:
Id prefer to name it `SetShowStatusBar` as this is not actually Showing it but just changing setting value
The same is for Hide.